packages/sst/src/runtime/handlers/node.ts

```diff
@@ -38,7 +38,6 @@ export const useNodeHandler = Context.memo(async () => {
           ),
           {
             env: {
-              ...process.env,
```

Comment:
@thdxr will this affect the spawned process?

Reply:
I _think_ this is the correct thing to. We shouldn't let your local env variable pollute right?